src/Evolution/Executables/GeneralizedHarmonic/EvolveGhSingleBlackHole.hpp

```diff
@@ -243,13 +253,40 @@ struct EvolutionMetavars : public GeneralizedHarmonicTemplateBase<3> {
         tmpl::map<tmpl::pair<gh_dg_element_array, dg_registration_list>>;
   };
 
+  using control_components =
+      control_system::control_components<EvolutionMetavars, control_systems>;
+
+  static void run_deadlock_analysis_simple_actions(
+      Parallel::GlobalCache<EvolutionMetavars>& cache,
+      const std::vector<std::string>& deadlocked_components) {
+    const auto& functions_of_time =
+        Parallel::get<::domain::Tags::FunctionsOfTime>(cache);
+
+    const std::string time_bounds =
+        ::domain::FunctionsOfTime::ouput_time_bounds(functions_of_time);
+
+    Parallel::printf("%s\n", time_bounds);
```

Comment:
Maybe have it say "Time bounds for functions of time are: "?

Reply:
`output_time_bounds()` actually already has that in the string it returns.

Reply:
Ah okay, sounds good! I'll merge as-is then :)